docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -310,6 +310,17 @@ otherwise the procedure will fail with similar message:
 ``Retention specified (1.00d) is shorter than the minimum retention configured in the system (7.00d)``.
 The default value for this property is ``7d``.
 
+drop_extended_stats
+~~~~~~~~~~~~~~~~~~~
+
+The ``drop_extended_stats`` command removes extended statistics from metadata file.
```

Comment:
from the metadata file. 

or 

from metadata files.

Reply:
also.. do we need to explain what extended stats includes. Or link to relevant info?